src/ray/raylet/raylet_client.h

```diff
@@ -24,14 +30,13 @@ class RayletConnection {
  public:
   /// Connect to the raylet.
   ///
-  /// \param raylet_socket The name of the socket to use to connect to the raylet.
-  /// \param worker_id A unique ID to represent the worker.
-  /// \param is_worker Whether this client is a worker. If it is a worker, an
-  ///        additional message will be sent to register as one.
-  /// \param driver_id The ID of the driver. This is non-nil if the client is a
-  ///        driver.
+  /// \param client_id A unique ID to represent the client.
+  /// \param raylet_socket The name of the socket for raylet connection.
+  /// \param num_retries The maximum num of retries to connect to the raylet socket.
+  /// \param timeout The timeout used to reconnect to the raylet socket.
```

Comment:
Make it something like `timeout_seconds` or `timeout_milliseconds` to indicate the units and document the units.